pyproject.toml

```diff
@@ -1,3 +1,44 @@
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools>=61.0"]
 build-backend = "setuptools.build_meta"
+
+[project]
+name = "pydisagg"
+version = "0.3.3"
+authors = [{ name = "IHME Math Sciences", email = "ihme.math.sciences@gmail.com" }]
+description = ""
+readme = "README.md"
+license = { text = "BSD 2-Clause License" }
+classifiers = [
+    "Development Status :: 4 - Beta",
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: BSD 2-Clause License",
```

Comment:
Unfortunately this is not in the list of the classifiers.
https://pypi.org/classifiers/
I think maybe we could stick with
```
License :: OSI Approved :: BSD License
```
As the parent classifier.